packages/sync-service/lib/electric/connection/manager.ex

```diff
@@ -883,12 +885,24 @@ defmodule Electric.Connection.Manager do
   end
 
   def handle_cast(
-        :consumers_ready,
+        {:consumers_ready, total_recovered, total_failed_to_recover},
         %State{
           current_phase: :connection_setup,
-          current_step: :waiting_for_consumers
+          current_step: {:waiting_for_consumers, start_time}
         } = state
       ) do
+    duration = System.monotonic_time() - start_time
+
+    Logger.notice(
+      "Consumers ready in #{System.convert_time_unit(duration, :native, :millisecond)}ms (#{total_recovered} shapes, #{total_failed_to_recover} failed to recover)"
+    )
+
+    :telemetry.execute(
+      [:electric, :connection, :consumers_ready],
+      %{duration: duration, total: total_recovered, failed_to_recover: total_failed_to_recover},
```

Comment:
```suggestion
      %{duration: duration, recovered: total_recovered, failed_to_recover: total_failed_to_recover},
```
The `total` keyword is a bit confusing w.r.t. to the failed ones (might think it's "total attempted")

Reply:
The whole metric is going to be `electric.connection.consumers_ready.total`, which reads to me very clearly as "total ready consumers", not "total attemted"